app/assets/stylesheets/framework_and_overrides.css.scss

```diff
@@ -102,3 +102,52 @@ img {
   height: 25vh;
   background-color: rgba(255,255,255,0.5);
 }
+
+.apartment-info {
+  background-color: #c5499b;
+  box-sizing: border-box;
+  margin: 5px 0;
+  display: block;
+  border-radius: 4px;
+  color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.